CODE_OF_CONDUCT.md

```diff
@@ -5,17 +5,16 @@ October 21, 2019**
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to make participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
-appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as community members,
+contributors committers, and project leaders pledge to make participation in our project and our
```

Comment:
Missing a coma here I guess (e.g. "contributors, committers")

Reply:
Mike has been notified about this typo! We will update this document once he provides us an updated document!

Thanks for catching this!